charts/posit-chronicle/values.yaml

```diff
@@ -0,0 +1,78 @@
+image:
+  repository: "ghcr.io/rstudio/chronicle"
+  tag: "latest"
+  imagePullPolicy: "Always"
+
+serviceaccount:
+  create: false
+  # -- Additional annotations to add to the chronicle-server serviceaccount
+  annotations: {
+    # EKS role arn example
+    # eks.amazonaws.com/role-arn: my-aws-iam-role-arn
+  }
+  # -- Additional labels to add to the chronicle-server serviceaccount
+  labels: {}
+
+service:
+  # -- The port to use for the REST service
+  port: 80
+  # -- Additional annotations to add to the chronicle-server service
+  annotations: {}
+  # -- Additional labels to add to the chronicle-server service
+  labels: {}
+  # -- Additional selector labels to add to the chronicle-server service
+  selectorLabels: {}
+
+# -- The number of replica pods to maintain for this service
+replicas: 1
+
+pod:
+  # -- The command and args to run in the chronicle-server container
+  command: "/chronicle"
+  args: ["start", "-c", "/opt/chronicle/config.gcfg"]
+  # -- Optional environment variables
+  env: []
+  # -- Additional annotations to add to the chronicle-server pods
+  annotations: {}
+  # -- Additional labels to add to the chronicle-server pods
+  labels: {}
+  # -- Additional selector labels to add to the chronicle-server pods
+  selectorLabels: {}
+  # -- A map used verbatim as the pod's "affinity" definition
+  affinity: {}
+  # -- A map used verbatim as the pod's "nodeSelector" definition
+  nodeSelector: {}
+  # -- An array used verbatim as the pod's "tolerations" definition
+  tolerations: []
+
+# We do want at least 1Gi of persistent volume space
+# to perform compaction/compression operations
+storage:
+  persistentVolumeSize: 1Gi
+
+config:
+  https:
+    # If https.enabled=true, we will ignore any http
+    # values and enable https in our config instead
+    enabled: false
+    key: ""
+    certificate: ""
+  localStorage:
+    enabled: true
+    location: "./chronicle-data"
+    retentionPeriod: "30d"
+  s3Storage:
+    enabled: false
+    bucket: "posit-chronicle"
+    prefix: ""
+    profile: ""
+    region: ""
+    compactionEnabled: false
+  metrics:
+    enabled: true
+  profiling:
+    enabled: false
```

Comment:
@atheriel I realized that i've been using pprof in our k8s environment, and plan on continuing to do so for some future QA. I would like to kep this!

Reply:
here's an example of me doing this:
```shell
 % kubectl port-forward chronicle-server-0 3030:3030
Forwarding from 127.0.0.1:3030 -> 3030
Forwarding from [::1]:3030 -> 3030
Handling connection for 3030
```

<img width="1275" alt="Screenshot 2023-11-21 at 12 30 32 PM" src="https://github.com/rstudio/helm/assets/18649003/fa20fb0f-1476-4b31-8e89-a87e83835d9a">

Reply:
Great! I didn't realize it was useful in its current form. Happy to keep it in then.